mobly/snippet/client_base.py

```diff
@@ -418,15 +418,9 @@ def handle_callback(self, callback_id, ret_value, rpc_func_name):
       The callback handler object.
     """
 
-  def stop_server(self):
-    """Proxy function of do_stop_server."""
-    self.log.debug('Stopping snippet %s.', self.package)
-    self.do_stop_server()
-    self.log.debug('Snippet %s stopped.', self.package)
-
   @abc.abstractmethod
-  def do_stop_server(self):
-    """Kills any running instance of the server."""
+  def stop(self):
+    """Stops the snippet server and releases all the requested resources."""
```

Comment:
Something like "releases all the resource acquired in `start_server`"?

Reply:
Done. Changed `start_server` to `initialize`.